apps/site/lib/site_web/templates/layout/_footer.html.eex

```diff
@@ -27,7 +27,7 @@
             <%= link "Press Releases", to: cms_static_page_path(@conn, "/news"), data: [scroll: true], class: "m-footer__link"  %>
             <%= link "Policies & Civil Rights", to: cms_static_page_path(@conn, "/policies"), data: [scroll: true], class: "m-footer__link"  %>
             <%= link "Safety", to: cms_static_page_path(@conn, "/safety") %>
-            <%= link "Performance", to: "http://www.mbtabackontrack.com/performance/index.html#/home", target: "_blank", class: "m-footer__link" %>
+            <%= link "Performance", to: "/performance-metrics", class: "m-footer__link" %>
```

Comment:
I was about to propose using the `cms_static_page_path` helper for this (as in the links immediately prior to this one) but I honestly couldn't identify a solid difference between the two approaches. 🤔